src/main/resources/MiLoG_Template.tex

```diff
@@ -3,241 +3,252 @@
 \usepackage{tabulary}
 \usepackage{graphicx}
 \usepackage{amssymb}
+\usepackage{array}
+\usepackage{background}
```

Comment:
Is background a standard package for all latex compilers? It worked for me but my tex client does not recognise this package, so I am not sure whether everybody can compile that file.

Reply:
It's included in TeX Live and MiKTeX (see https://ctan.org/pkg/background), so I suppose it is somewhat of a standard package.